azure-iot-hub-devicesdk/azure/iot/hub/devicesdk/sync_clients.py

```diff
@@ -88,6 +88,9 @@ def __init__(self, transport):
         """
         super(GenericClientSync, self).__init__(transport)
         self._inbox_manager = InboxManager(inbox_type=SyncClientInbox)
+        self._transport.on_transport_method_call_message_received = (
+            self._inbox_manager.route_method_call
+        )
```

Comment:
not familiar enough with python to understand why these parenthesis are necessary... educate me?

Reply:
This isn't my choice, it's black.

The line was over 100 characters. So black formatted it to use a different line. In doing so, it thus requires parenthesis  to keep it together (or explicit line breaks with '\', but black seems to shy away from those)

---
In reply to: [266185502](https://github.com/Azure/azure-iot-sdk-python-preview/pull/61#discussion_r266185502) [](ancestors = 266185502)

Reply:
oh ok. thanks, i'm educated now :)

---
In reply to: [266598278](https://github.com/Azure/azure-iot-sdk-python-preview/pull/61#discussion_r266598278) [](ancestors = 266598278,266185502)